appcelerator/titanium_mobile
# [TIMOB-25733] Android: Fixed bug where view with border may fail to appear when sized larger than screen.

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25733

**Summary:**
- This bug was introduced in Titanium 7.0.2.RC.
- Fixed by reverting [TIMOB-25661](https://jira.appcelerator.org/browse/TIMOB-25661).
  * This is a Google bug and we don't have good work-around yet which doesn't break other things.

**Test:**
1. Acquire a Google Pixel XL phone which is running Android 7.0 or higher.
2. Take a picture with it's camera.
3. Build and run the below code.
4. On app startup, select the photo you just took.
5. Verify that the camera shot was displayed correctly onscreen.
6. Verify that you do not see the following error message in the Android log.

> View: TiBorderWrapperView not displayed because it is too large to fit into a software layer (or drawing cache)

---
**Code:**

```javascript
var window = Ti.UI.createWindow();
window.addEventListener("open", function(e) {
	Ti.Media.openPhotoGallery(
	{
		allowMultiple: false,
		allowEditing: false,
		autohide: true,
		mediaTypes: [Ti.Media.MEDIA_TYPE_PHOTO],
		success: function(e) {
			if (e.mediaType === Ti.Media.MEDIA_TYPE_PHOTO) {
				var scrollView = Ti.UI.createScrollView(
				{
					layout: "vertical",
					scrollType: "vertical",
					showHorizontalScrollIndicator: false,
					shorVerticalScrollIndicator: true,
				});
				scrollView.backgroundColor = "blue";
				scrollView.add(Ti.UI.createImageView(
				{
					borderRadius: window.size.width / 2,
					borderWidth: "20dp",
					borderColor: "red",
					image: e.media,
					width: Ti.UI.SIZE,
					height: Ti.UI.SIZE,
				}));
				scrollView.add(Ti.UI.createLabel({ text: "Bottom Text" }));
				window.add(scrollView);
			}
		},
		error: function() {
			Ti.API.info("@@@ Photo gallery selection error.");
		},
	});
});
window.open();
```
